pkg/management/authorization.go

```diff
@@ -478,6 +481,39 @@ func (c *authorizationComponentMW) UpdateRealmCustomConfiguration(ctx context.Co
 	return c.next.UpdateRealmCustomConfiguration(ctx, realmName, customConfig)
 }
 
+func (c *authorizationComponentMW) GetRealmBackOfficeConfiguration(ctx context.Context, realmName string, groupID string) (api.BackOfficeConfiguration, error) {
+	var action = MGMTGetRealmBackOfficeConfiguration.String()
+	var targetRealm = realmName
+
+	if err := c.authManager.CheckAuthorizationOnTargetRealm(ctx, action, targetRealm); err != nil {
```

Comment:
As we provide a groupID, I think we could have a CheckAuthorizationOnTargetGroupID
If we change it, we should not forget to adapt scope in the Action definition